src/routes/ApiRoutes.ts

```diff
@@ -275,9 +284,14 @@ router.post(`/notifications/register`, redirect.register)
  * @apiParam (Request body) {Number[]} networks Array of network indexes (https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
  * If specified empty array, unsubscribe will happen on all supported networks
  *
- * @apiParamexample {json} Request Body Example
+ * @apiParamexample {json} Request Body Example for unsubscribe iOS device
  * {
  *      "deviceID": "B14BD907-324A-4A40-98A1-A255C26D2BE52",
+ *      "networks": [60, 61]
```

Comment:
networks would be always `[]` for now! don't handle it

Reply:
OK, made changes